Sources/App/Controllers/UsersController.swift

```diff
@@ -0,0 +1,54 @@
+import Vapor
+import HTTP
+
+final class UsersController: ResourceRepresentable {
```

Comment:
I think by buying into the Vapor system past routing, we're adding a level of complexity that's pretty unnecessary. This seems to work alright though. 👍